### PR TITLE
CMR-4204

### PR DESCRIPTION
**Pull request by ygliuvt**
Fixed variable association to work with the variable native-id change.
Cleaned up code related variable metadata in UMM JSON format in metadata-db.
Change services metadata from EDN to UMM JSON in metadata will be done in a separate PR.

### CODE REVIEW

File: metadata-db-app/src/migrations/060_update_var_association_var_name_length.clj [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "Migrates the database up to version 60."
+  []
+  (println "migrations.060-update-var-association-var-name-length up...")
+  ;; The variable_name field actually saves the native-id of the variable in the association
+  ;; It is temporary. We didn't change the name of the field to native_id as we will eventaully
+  ;; change this to concept_id later as described in CMR-4254
+  (h/sql "alter table METADATA_DB.cmr_variable_associations modify variable_name VARCHAR(1030)"))
+
+
+(defn down
```
Comment (laurenfrederick) [inline metadata-db-app/src/migrations/060_update_var_association_var_name_length.clj]: Will this truncate variable names that are more than 80 characters?

Comment (ygliuvt) [inline metadata-db-app/src/migrations/060_update_var_association_var_name_length.clj]: Good point. I updated the migration to add the truncation of the field value.

Comment (laurenfrederick) [inline metadata-db-app/src/migrations/060_update_var_association_var_name_length.clj]: Cool. I don't see the change yet, but I'll approve pending that commit.

Comment (ygliuvt) [inline metadata-db-app/src/migrations/060_update_var_association_var_name_length.clj]: It is pushed now.

**Pull request merged**